credoai/reporting/dataset_base.py

```diff
@@ -116,30 +140,46 @@ def create_report(self, filename=None):
 
         self.figs.append(fig)
 
-        # Generate group difference barplots
-        results = results_all["group_diffs"]
+    def _plot_group_diff(self):
+        """ Generates group difference barplots
+        """
+        results = self.results_all["group_diffs"]
+        fig, axs = plt.subplots(nrows=len(results), dpi=150)
+        i = 0
         for k, v in results.items():
             df = pd.DataFrame(v.items(), columns=["feature", "group difference"])
-            fig = plt.figure(figsize=(8, 4), dpi=150)
-            ax = sns.barplot(
-                x="feature",
-                y="group difference",
-                data=df,
-                palette=plot_utils.credo_diverging_palette(1),
-                alpha=1,
-            )
+            if len(results) > 1:
```

Comment:
This works even if len(results) == 1 - no need for conditional logic

Reply:
That's also what I though but threw out `TypeError: 'AxesSubplot' object is not subscriptable`. Tried to fixed it with parameters, but not success. So opted for this solution.